diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8a48bef66..5c54f80b5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,6 +3,13 @@ cmake_minimum_required(VERSION 3.1.3...3.29.2 FATAL_ERROR)
 
 # Declare project
 project(DCMTK)
+
+# indra required flags for emscripten
+string(TOLOWER ${CMAKE_SYSTEM_NAME} AL_SYSTEM)
+if ("${AL_SYSTEM}" STREQUAL "emscripten")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wdeprecated-declarations -pthread -s USE_PTHREADS -s USE_LIBPNG -s USE_LIBJPEG -s USE_FREETYPE -fexceptions")
+endif()
+
 if (NOT DEFINED CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD "11" CACHE STRING "Set C++ standard to be used (98,11,14,17,20)")
 else()
diff --git a/ofstd/libsrc/CMakeLists.txt b/ofstd/libsrc/CMakeLists.txt
index 508189b65..37f0f83b5 100644
--- a/ofstd/libsrc/CMakeLists.txt
+++ b/ofstd/libsrc/CMakeLists.txt
@@ -27,7 +27,6 @@ DCMTK_ADD_LIBRARY(ofstd
   ofmath.cc
   ofsockad.cc
   ofrand.cc
-  ofwhere.c
 )
 
 DCMTK_TARGET_LINK_LIBRARIES(ofstd config ${CHARSET_CONVERSION_LIBS} ${SOCKET_LIBS} ${THREAD_LIBS} ${WIN32_STD_LIBRARIES})
diff --git a/ofstd/libsrc/Makefile.in b/ofstd/libsrc/Makefile.in
index bf2811e28..f44f60c01 100644
--- a/ofstd/libsrc/Makefile.in
+++ b/ofstd/libsrc/Makefile.in
@@ -20,7 +20,7 @@ LOCALDEFS =
 objs = oflist.o ofstring.o ofcmdln.o ofconapp.o offname.o ofconsol.o ofthread.o \
 	ofcond.o ofstd.o ofcrc32.o ofdate.o oftime.o ofdatime.o oftimer.o \
 	ofconfig.o ofchrenc.o oftempf.o ofxml.o ofuuid.o offile.o offilsys.o \
-	ofmath.o oferror.o ofsockad.o ofrand.o ofstrutl.o ofipc.o ofwhere.o
+	ofmath.o oferror.o ofsockad.o ofrand.o ofstrutl.o ofipc.o
 
 library = libofstd.$(LIBEXT)
 
@@ -46,6 +46,6 @@ distclean:
 
 
 dependencies:
-	$(CXX) -MM $(defines) $(includes) $(CPPFLAGS) $(CXXFLAGS) *.cc *.c > $(DEP)
+	$(CXX) -MM $(defines) $(includes) $(CPPFLAGS) $(CXXFLAGS) *.cc > $(DEP)
 
 include $(DEP)
